docs/usage/templates.md

```diff
@@ -30,3 +30,11 @@ If you want to print pretty JSON with Handlebars you can use the built-in functi
 `{{{stringToPrettyJSON myvar}}}`
 
 In the example above `myvar` is a variable/field, that contains valid JSON.
+
+### encodeURIComponent
+
+If you want to ..., use the built-in function `encodeURIComponent` like this:
+
+### replace
+
+If you want to ..., use the built-in function `replace` like this:
```

Comment:
I also need help to explain what `replace` does, and need an example on how to use it.

Reply:
I think this is the JavaScript `string.replace` function

Reply:
We pass the first arg to Regex constructor with global flag set. Then we pass that to replace function with second arg as replacement value.

Reply:
Need to check if it's replace or replace all

Reply:
I think replace all, because of the global flag

https://github.com/renovatebot/renovate/blob/98e702909011d2079d6d50eab372eb6d5b3e38f2/lib/util/template/index.ts#L9-L13

Reply:
Let's make sure that's captured in the docs then